portal-impl/src/portal.properties

```diff
@@ -6972,6 +6977,11 @@
     #
     feature.flag.LPD-29516.type=beta
 
+    #
+    # Env: LIFERAY_FEATURE_PERIOD_FLAG_PERIOD__UPPERCASEL__UPPERCASEP__UPPERCASED__MINUS__NUMBER3__NUMBER0__NUMBER3__NUMBER7__NUMBER1__PERIOD_TYPE
+    #
+    feature.flag.LPD-30371.type=beta
```

Comment:
Please make this a dev feature flag. We should only switch it to beta after product reviews the development in master.